### PR TITLE
Changed setting order and label names.

### PR DESCRIPTION
**Pull request by PMQABen**
Fixes https://processmaker.atlassian.net/browse/FOUR-2191

### CODE REVIEW

File: resources/js/processes/screen-builder/components/file-upload-control.js [hunk 1]
```diff
@@ -17,28 +17,28 @@ export default {
...
       },
       {
         type: "FormInput",
-        field: "name",
+        field: "accept",
         config: {
-          label: "Name",
-          name: 'Name',
-          helper: "The name of the upload",
-          validation: 'regex:/^(?:[A-Z_.a-z])(?:[0-9A-Z_. \/a-z])*$/|required'
```
Comment (velkymx) [inline resources/js/processes/screen-builder/components/file-upload-control.js:41]: I think we should keep the validation...

**Review by velkymx — CHANGES_REQUESTED** [review]
I think we should keep the validation. Having examples of the mime-times is a good step though!!!

File: resources/js/processes/screen-builder/components/file-upload-control.js (hunk 1 above)
Comment (PMQABen) [reply resources/js/processes/screen-builder/components/file-upload-control.js:41]: The validation for "Name" is being kept, we're renaming it to "Variable Name" and moving it to the top to match how the rest of the screen builder configs are laid out.

Comment (PMQABen) [reply resources/js/processes/screen-builder/components/file-upload-control.js:41]: image

Comment (velkymx) [reply resources/js/processes/screen-builder/components/file-upload-control.js:41]: Great

**Pull request merged**